cli/borges/consumer.go

```diff
@@ -106,9 +112,10 @@ type consumerOpts struct {
 	queueOpts
 	metricsOpts
 
-	Locking string `long:"locking" env:"BORGES_LOCKING" default:"local:" description:"locking service configuration"`
-	Workers int    `long:"workers" env:"BORGES_WORKERS" default:"1" description:"number of workers, 0 means the same number as processors"`
-	Timeout string `long:"timeout" env:"BORGES_TIMEOUT" default:"10h" description:"deadline to process a job"`
+	Locking        string `long:"locking" env:"BORGES_LOCKING" default:"local:" description:"locking service configuration"`
+	LockingTimeout string `long:"locking-timeout" env:"BORGES_LOCKING_TIMEOUT" default:"0" description:"timeout to acquire lock, 0 means no timeout"`
+	Workers        int    `long:"workers" env:"BORGES_WORKERS" default:"1" description:"number of workers, 0 means the same number as processors"`
+	Timeout        string `long:"timeout" env:"BORGES_TIMEOUT" default:"10h" description:"deadline to process a job"`
```

Comment:
10h - wow!

Reply:
Some repos can take a lot of time. We use smaller values for our deployment.